tensorflow_probability/examples/bayesian_neural_network.py

```diff
@@ -216,27 +206,38 @@ def main(argv):
   if FLAGS.fake_data:
     mnist_data = build_fake_data()
   else:
-    mnist_data = mnist.read_data_sets(FLAGS.data_dir)
+    mnist_data = mnist.read_data_sets(FLAGS.data_dir, reshape=False)
 
   with tf.Graph().as_default():
     (images, labels, handle,
      training_iterator, heldout_iterator) = build_input_pipeline(
          mnist_data, FLAGS.batch_size, mnist_data.validation.num_examples)
 
-    # Build a Bayesian neural net. We use the Flipout Monte Carlo estimator for
-    # each layer: this enables lower variance stochastic gradients than naive
-    # reparameterization.
+    # Build a Bayesian LeNet5 network. We use the Flipout Monte Carlo estimator for
+    # the convolution and fully-connected layers: this enables lower variance stochastic
+    # gradients than naive reparameterization.
     with tf.name_scope("bayesian_neural_net", values=[images]):
-      neural_net = tf.keras.Sequential()
-      for units in FLAGS.layer_sizes:
-        layer = tfp.layers.DenseFlipout(
-            units,
-            activation=FLAGS.activation)
-        neural_net.add(layer)
-      neural_net.add(tfp.layers.DenseFlipout(10))
-      logits = neural_net(images)
-      labels_distribution = tfd.Categorical(logits=logits)
-
+        neural_net = tf.keras.Sequential([
+                tfp.layers.Convolution2DFlipout(
```

Comment:
Following our style guide, we use PEP-8. Namely, use only 4 spaces when indenting arguments that don't fit in one line. Alternatively, write one kwarg per line and align their whitespace, e.g.,

```python
y = f(x,
      kwarg1=kwarg1,
      kwarg2=kwarg2)
```